lib/kitchen/provisioner/habitat.rb

```diff
@@ -51,8 +52,12 @@ class Habitat < Base
       default_config :user_toml_name, "user.toml"
       default_config :override_package_config, false
 
-      # experimental
-      default_config :use_screen, false
```

Comment:
Why was the decision made to remove screen support? I'm fine with it since it was experimental and this is < 1.x but I just want to understand it. No screen support on Windows?

Reply:
It didn't work for me and I don't believe it works for Windows so I removed it. I'm happy to revisit that change if someone with more experience with screen wants to chat about it.

Reply:
is the the same as the `gui: false` option for vagrant?

If so, we normally use it to save on procession power on windows. 

```yaml
driver:
  name: vagrant
  customize:
    memory: 4096
  boot_timeout: 2400
  gui: false
```

Reply:
@sam1el screen is a utility to split terminals in a single ssh session.